x/dex/types/keys.go

```diff
@@ -247,89 +247,115 @@ func LimitOrderExpirationKey(
+	WithdrawEventReserves1Withdrawn = "Reserves1Withdrawn"
+	WithdrawEventSharesRemoved      = "SharesRemoved"
 )
 
+// Swap Event Attributes
 const (
-	SwapEventKey      = "NewSwap"
+	SwapEventKey      = "Swap"
 	SwapEventCreator  = "Creator"
 	SwapEventReceiver = "Receiver"
+	SwapEventToken0   = "Token0"
+	SwapEventToken1   = "Token1"
 	SwapEventTokenIn  = "TokenIn"
 	SwapEventTokenOut = "TokenOut"
 	SwapEventAmountIn = "AmountIn"
 	SwapEventAmoutOut = "AmountOut"
 )
 
+// Multihop-Swap Event Attributes
 const (
-	MultihopSwapEventKey      = "NewMultihopSwap"
-	MultihopSwapEventCreator  = "Creator"
-	MultihopSwapEventReceiver = "Receiver"
-	MultihopSwapEventCoinIn   = "CoinIn"
-	MultihopSwapEventCoinOut  = "CoinOut"
-	MultihopSwapEventRoute    = "Route"
+	MultihopSwapEventKey       = "MultihopSwap"
+	MultihopSwapEventCreator   = "Creator"
+	MultihopSwapEventReceiver  = "Receiver"
+	MultihopSwapEventTokenIn   = "TokenIn"
+	MultihopSwapEventTokenOut  = "TokenOut"
+	MultihopSwapEventAmountIn  = "AmountIn"
+	MultihopSwapEventAmountOut = "AmountOut"
+	MultihopSwapEventRoute     = "Route"
 )
 
+// Place LimitOrder Event Attributes
 const (
-	PlaceLimitOrderEventKey        = "NewPlaceLimitOrder"
+	PlaceLimitOrderEventKey        = "PlaceLimitOrder"
 	PlaceLimitOrderEventCreator    = "Creator"
 	PlaceLimitOrderEventReceiver   = "Receiver"
+	PlaceLimitOrderEventToken0     = "Token0"
+	PlaceLimitOrderEventToken1     = "Token1"
 	PlaceLimitOrderEventTokenIn    = "TokenIn"
 	PlaceLimitOrderEventTokenOut   = "TokenOut"
 	PlaceLimitOrderEventAmountIn   = "AmountIn"
+	PlaceLimitOrderEventLimitTick  = "LimitTick"
+	PlaceLimitOrderEventOrderType  = "OrderType"
 	PlaceLimitOrderEventShares     = "Shares"
 	PlaceLimitOrderEventTrancheKey = "TrancheKey"
 )
 
+// Withdraw LimitOrder Event Attributes
 const (
-	WithdrawFilledLimitOrderEventKey        = "NewWithdraw"
+	WithdrawFilledLimitOrderEventKey        = "Withdraw"
 	WithdrawFilledLimitOrderEventCreator    = "Creator"
+	WithdrawFilledLimitOrderEventToken0     = "Token0"
+	WithdrawFilledLimitOrderEventToken1     = "Token1"
 	WithdrawFilledLimitOrderEventTokenIn    = "TokenIn"
 	WithdrawFilledLimitOrderEventTokenOut   = "TokenOut"
 	WithdrawFilledLimitOrderEventTrancheKey = "TrancheKey"
 	WithdrawFilledLimitOrderEventAmountOut  = "AmountOut"
 )
 
+// Cancel LimitOrder Event Attributes
 const (
-	GoodTilPurgeHitGasLimitEventKey = "GoodTilPurgeHitGasLimit"
-	GoodTilPurgeHitGasLimitEventGas = "Gas"
-)
-
-const (
-	CancelLimitOrderEventKey        = "NewWithdraw"
+	CancelLimitOrderEventKey        = "Withdraw"
 	CancelLimitOrderEventCreator    = "Creator"
+	CancelLimitOrderEventToken0     = "Token0"
+	CancelLimitOrderEventToken1     = "Token1"
 	CancelLimitOrderEventTokenIn    = "TokenIn"
 	CancelLimitOrderEventTokenOut   = "TokenOut"
 	CancelLimitOrderEventTrancheKey = "TrancheKey"
 	CancelLimitOrderEventAmountOut  = "AmountOut"
 )
 
+// Tick Update Event Attributes
+const (
+	EventTypeTickUpdate       = "TickUpdate"
```

Comment:
this line seems very inconsistent with all the other emitted events we have.

the event "type" should probably be "message" not "TickUpdate" (even though we don't sign a TickUpdate Msg), the value "TickUpdate" is already included in the attributes under the "action" key.

I believe the event types are intended to be a specific list of known values to Cosmos SDK, not any arbitrary test. But I may be wrong about that. [EventTypeMessage](https://docs.cosmos.network/v0.46/core/events.html#typed-events)